### PR TITLE
Write the readme

### LINKED ISSUE #102: Write the readme
## Description 📹

**Write a readme. You can take some ideas from there: https://github.com/lfglabs-dev/api.starknet.quest/blob/testnet/README.md**

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
- [ ] Add each section of the readme, such as: How to install it
- [ ] How to run it
- [ ] Troubleshoot section

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by Benjtalkshow**
Closes #102

### CODE REVIEW

**Comment by Benjtalkshow** [comment]
@Marchand-Nicolas 
Please Review

File: README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+6. **Pull Request Steps**:
+   - Fork the repository
+   - Create a feature branch
+   - Commit your changes
+   - Push to your branch
+   - Create a Pull Request
+
+Thank you for your contribution! 🙏
+
+⚠️ **WARNING**: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.
```
Comment (Marchand-Nicolas) [inline README.md]: You can remove this warning, because it is specific to OD

File: README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# StarknetID API
+
+StarknetID API is a Rust-based service that provides API endpoints for interacting with StarkNet identities. This API serves as the backend infrastructure for StarkNet identity management and resolution.
+
+## Description
+
+The StarknetID API is designed to handle identity-related operations on the StarkNet blockchain. It provides a robust interface for managing and querying StarkNet identities, making it easier to integrate identity services into applications built on StarkNet.
```
Comment (Marchand-Nicolas) [inline README.md]: You can remove the first sentence (The StarknetID API is designed to handle identity-related operations on the StarkNet blockchain.)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# StarknetID API
+
+StarknetID API is a Rust-based service that provides API endpoints for interacting with StarkNet identities. This API serves as the backend infrastructure for StarkNet identity management and resolution.
```
Comment (Marchand-Nicolas) [inline README.md]: You can remove this, because it's redundant with the description

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

**Comment by Benjtalkshow** [comment]
@Marchand-Nicolas Done Ser

**Review by Marchand-Nicolas — APPROVED** [review]
Lgtm well done!

**Pull request merged**